managing_providers/_topics/automation_management_providers.md

```diff
@@ -365,3 +365,174 @@ To use the button to run an Ansible Tower job on a virtual machine:
 If you selected a service dialog to run when creating the button, {{ site.data.product.title_short }} will then prompt you to enter variables to complete the task. After entering your desired parameters, {{ site.data.product.title_short }} takes you to the **Requests** page.
 
 The service item’s details can be viewed in menu: **Services > My Services** in {{ site.data.product.title_short }}.
+
+## OpenTofu
+
+OpenTofu is an open source infrastructure as code tool, which can be used to build, change, and version the infrastructure. You can use OpenTofu to define infrastructure resources in human-readable configuration files that you can use to version, reuse, and share.
+
+OpenTofu is built into {{ site.data.product.title_short }} so you do not need to install any additional components. 
+
+If you want to use the Embedded Terraform feature in {{ site.data.product.title_short }} that is deployed as a virtual machine appliance, then you need to manually import the Terraform image on an appliance:
+
+## Importing OpenTofu image on an appliance
+
+**Note**: Follow this section if you have {{ site.data.product.title_short }} that is deployed as a virtual machine appliance. These steps are not applicable to {{ site.data.product.title_short }} that is deployed as a containerized deployment (podified).
+
+Use the following command to import the OpenTofu image on your appliance server.
+
+``` bash
+runuser --login manageiq --command 'podman --root=/var/www/miq/vmdb/data/containers/storage image load --input /tmp/<OpenTofu_image>'
+```
+
+Where `OpenTofu_image` is the name of your OpenTofu image.
+
+``` bash
+runuser --login manageiq --command 'podman --root=/var/www/miq/vmdb/data/containers/storage image load --input /tmp/infra-mgmt-opentofu-runner_PR-110-204033d-20240521-x86_64.tar'
+```
+
+You also need to set the docker image name in advanced settings page before enabling the server role. Navigate to the **Settings** > **Application Settings** in {{ site.data.product.title_short }} UI and set the value for `container_image` field.
+
+An example value of this field is `container_image: docker.io/manageiq/opentofu-runner:latest`.
+
+## Usage of Embedded Terraform (OpenTofu)
+
+The following sections show the usage of Embedded Terraform in {{ site.data.product.title_short }}. The following sections apply to {{ site.data.product.title_short }} that is deployed as a containerized deployment (podified) and {{ site.data.product.title_short }} that is deployed as a virtual machine appliance:
+
+1. Enable the Embedded Terraform server role.
+2. Add a source control repository that contains your templates.
+3. Add credentials for your cloud providers.
+4. Create a Service Catalog item with the wanted Terraform template.
+
+### Enabling the Embedded Terraform Server Role
+
+In {{ site.data.product.title_short }}, the Embedded Terraform server role is disabled by default. Enable this server role to use Terraform Automation Inside.
```

Comment:
Could we use "Embedded Terraform (OpenTofu)" instead of "Terraform Automation Inside" ?  I assume this came from Ansible Inside but we don't call it this anywhere else